### DISCUSSION TITLE
模板发布流程 Bug 提交区

### DISCUSSION THREAD

**Discussion by a76yyyy** [qd-today/templates#344] [General]
### 请在评论中说明以下内容：
1. Bug 的具体表现为怎样？
2. 详细说明如何复现这个 Bug；
3. Bug 对应的具体 [Actions](https://github.com/qd-today/templates/actions) 编号

> **请在反馈前先浏览评论区, 确认无相关 Bug, 切勿重复提交 Bug**

**Comment by chen8945** [comment]
https://github.com/qd-today/templates/actions/runs/6887073655
应该算是历史遗留问题吧，以前靠 PR 发布模板的时候有两个模板是指向同一个评论区，在修改其中一个之后，Actions 不会工作
![image](https://github.com/qd-today/templates/assets/44148812/3fc65317-e218-47af-ab01-18405f17279d)

之前也碰到过一次这样的问题，那次的操作是先手动到需要删除的模板按格式填好之后再打上 `invalid` 标签，再删除完成之后再进行新的更新提交
https://github.com/qd-today/templates/pull/373#issuecomment-1609098975
![image](https://github.com/qd-today/templates/assets/44148812/6856ca74-5b06-4204-8a37-2253057c779a)

**Reply by a76yyyy** [reply]
![Screenshot_2023-11-17-08-13-00-717_com.android.browser-edit.jpg](https://github.com/qd-today/templates/assets/56478790/6ab112d3-8321-40a6-82e1-71a50096d5bd)

你需要在label中加入har标签才会执行action

此外，一个评论区根据模板内容中的模板名唯一对应一个har，指向同一个评论区的多个har只会修改同名har的内容

**Comment by chen8945** [comment]
> 你需要在label中加入har标签才会执行action

这就很尴尬了，改完忘了这茬，忘记以前模板是没标签的了

> 此外，一个评论区根据模板内容中的模板名唯一对应一个har，指向同一个评论区的多个har只会修改同名har的内容

关于这个是因为以前还没有模板发布流程的时候，为了方便反馈管理，所以将同一个网站的两个har指向了同一个评论区
